tests-hivemq-platform-operator/src/integrationTest/java/com/hivemq/helmcharts/single/HelmVolumeClaimTemplatesIT.java

```diff
@@ -0,0 +1,106 @@
+package com.hivemq.helmcharts.single;
+
+import com.hivemq.helmcharts.AbstractHelmChartIT;
+import io.fabric8.kubernetes.api.model.PersistentVolumeClaim;
+import io.fabric8.kubernetes.api.model.Quantity;
+import io.fabric8.kubernetes.api.model.Volume;
+import io.fabric8.kubernetes.api.model.VolumeMount;
+import org.jetbrains.annotations.NotNull;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Timeout;
+
+import java.time.Duration;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
+
+@Tag("Volumes")
+class HelmVolumeClaimTemplatesIT extends AbstractHelmChartIT {
```

Comment:
We discussed that it would be beneficial to add some asserts that the PersistentVolumes and the VolumeClaimTemplates are working correctly (in comparison to normal Volumes).

So maybe add assertions that each Pod has its own PV and that they are re-mounted when the Pod restarts.

Reply:
done